#--- vym-1.7.0/mainwindow.cpp~	2005-07-11 22:55:32.000000000 +0300
#+++ vym-1.7.0/mainwindow.cpp	2005-08-09 09:24:19.000000000 +0300
#@@ -2322,7 +2322,7 @@
# 		docpath="./vym.app/Contents/vym.pdf";
# 	#else
# 		// default path in SUSE LINUX
#-		docpath="/usr/share/doc/packages/vym/doc/vym.pdf";
#+		docpath= DOCPATH "/vym.pdf";
# 	#endif
# 
# 	if (!QFile (docpath).exists() )
--- vym-1.7.0/mainwindow.cpp~	2005-08-09 09:39:04.000000000 +0300
+++ vym-1.7.0/mainwindow.cpp	2005-08-09 09:40:03.000000000 +0300
@@ -2321,8 +2321,8 @@
 	#if defined(Q_OS_MACX)
 		docpath="./vym.app/Contents/vym.pdf";
 	#else
-		// default path in SUSE LINUX
-		docpath="/usr/share/doc/packages/vym/doc/vym.pdf";
+		// path in PLD Linux
+		docpath="/usr/share/vym/vym.pdf";
 	#endif
 
 	if (!QFile (docpath).exists() )
--- vym-1.7.0/mainwindow.cpp~	2005-08-09 09:42:16.000000000 +0300
+++ vym-1.7.0/mainwindow.cpp	2005-08-09 09:52:46.000000000 +0300
@@ -189,7 +189,7 @@
 		// application to open PDFs
 		p="/vym/mainwindow/readerPDF";
 		#if defined(Q_OS_LINUX)
-			s=settings.readEntry (p,"acroread");
+			s=settings.readEntry (p,"kpdf");
 		#else
 			#if defined(Q_OS_MACX)
 				s=settings.readEntry (p,"/usr/bin/open");
